models/applications.js

```diff
@@ -6,9 +6,9 @@ module.exports = function(sequelize, DataTypes) {
 			primaryKey: true,
 			defaultValue: DataTypes.UUIDV4
 		},
-		control_number: DataTypes.INTEGER, //eslint-disable-line camelcase
-		form_type: DataTypes.STRING, //eslint-disable-line camelcase
-		website_addr: DataTypes.STRING, //eslint-disable-line camelcase
```

Comment:
need to exclude camelcase linting from models directory - false positive due to database naming convention